clients/src/main/java/org/apache/kafka/clients/admin/DeleteConsumerGroupsResult.java

```diff
@@ -32,9 +31,9 @@
  */
 @InterfaceStability.Evolving
 public class DeleteConsumerGroupsResult {
-    private final Map<CoordinatorKey, KafkaFutureImpl<Void>> futures;
+    private final Map<CoordinatorKey, KafkaFuture<Void>> futures;
 
-    DeleteConsumerGroupsResult(final Map<CoordinatorKey, KafkaFutureImpl<Void>> futures) {
+    DeleteConsumerGroupsResult(final Map<CoordinatorKey, KafkaFuture<Void>> futures) {
```

Comment:
Should we also remove CoordinatorKey here to remain consistent? This is less important because this constructor is not public.

Reply:
+ 1